### ISSUE TITLE
Where does the Moods come from?

### ISSUE THREAD

**Issue by kronzo** [michaelherger/lms-plugin-tidal#8]
Where do these five German moods come from? I switched my LMS, my Mac OS X and TIDAL to English, deleted my account in the plugin and reactivated it again. But it didn't change anything. The TIDAL app also shows a much larger selection of moods.

![Screenshot at Feb 12 20-58-55](https://github.com/michaelherger/lms-plugin-tidal/assets/6114202/bbfed41f-bfff-4d72-8276-f56e3b1e8b09)

**Comment by michaelherger** [comment]
Yeah, I'm seeing the same. The web UI seems to be using a custom endpoint to get them, not the regular API's. And the Python implementations seems to have switched to that a while ago (see https://github.com/tamland/python-tidal/blob/b8c2c2de53ec2151b8073e4a7ab2cb0461b35b40/tidalapi/session.py#L951). Maybe we'll have to do the same. But I prefer to stick with documented functions...
